source/general/wikicontributing.rst

```diff
@@ -7,61 +7,63 @@ Contributing to the Wiki
 Setting up the environment
 ==========================
 
-To begin editing the wiki we first need to set our environment up. The prerequisites needed is just `python <https://www.python.org/downloads/>`_.
+To begin editing the Wiki we first need to up our environment. `Python`_ is the only dependency required.
 
-.. note:: When installing python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
+.. note:: When installing Python make sure too tick the ``Add Python to PATH`` option to be able to use python within your terminal/command prompt
 
-First thing you need to do is to fork the git repository and then clone it.
+
+1. Fork the GitHub repository and clone it.
 
 .. note:: Remember to replace *exampleusername* with your actual github username
 
 .. code:: bash
 
     git clone https://github.com/exampleusername/ProjectKorraWiki.git
 
-The wiki is made with sphinx using the *sphinx_rtd_theme* from readthedocs. We've bundled all you need into one file.
-Navigate to the directory you cloned the project to, then we will install all the requirements by running the following command in terminal / command prompt
+The wiki is built with Sphinx using the *sphinx_rtd_theme* from ReadTheDocs.
+Navigate to the directory you cloned the project to, then it will install all the requirements by running the following command in terminal / command prompt.
 
 .. code:: bash
 
     pip install -r .\requirements.txt
 
-This should install both sphinx as well as the theme.
+This should install both Sphinx as well as the theme.
 
-To check that sphinx is correctly install run:
+To check that Sphinx is correctly installed run:
 
 .. code:: bash
 
     sphinx-build -version
 
-If sphinx has been installed correctly it should output sphinx-build and the version number For example ``sphinx-build 1.7.5``
+If Sphinx has been installed correctly it should output sphinx-build and the version number For example ``sphinx-build 1.7.5``
 
-The reason we need sphinx is so you can view your changes locally before you submit your changes as a pull request.
+Sphinx is required to view your changes locally before submitting them.
 
-Viewing your changes locally
-============================
+Viewing Local Changes
+=====================
 
-Once you have made some changes and you are ready to see your edits locally open terminal / command prompt and run
+Once you have made some changes and are ready to see your edits locally, open terminal / command prompt and run
 
 .. code:: bash
 
     .\make.bat html
 
-This should build a copy of the wiki into the build folder in the root directory of your repository.
-To view the wiki double click on ``build > html > index.html``
```

Comment:
Maybe leave this code block path?

Reply:
Yeah, especially if someone doesn't know what they're doing, this is an easy way to learn.

Reply:
You should leave the code path.